scripts/globals/bcnm.lua

```diff
@@ -1926,15 +1955,28 @@ end
 -- onTrigger Action
 -----------------------------------
 xi.bcnm.onTrigger = function(player, npc)
+    local dolog = player:getZoneID() == 139 or player:getZoneID() == 140
+
```

Comment:
Yeah, also this dolog and all the dolog conditions are not needed as they are just for internal debugging that we were doing on horizon.

Reply:
Removed